owiddata/generate-owiddata-stats.sh

```diff
@@ -9,9 +9,8 @@ OWID_COMMIT_JSON=$(curl -sS "https://api.github.com/repos/owid/covid-19-data/com
 export OWID_COMMIT_SHA=$(echo $OWID_COMMIT_JSON | python -c "import sys, json; print(json.load(sys.stdin)[0]['sha'])")
 export OWID_COMMIT_DATE=$(echo $OWID_COMMIT_JSON | python -c "import sys, json; print(json.load(sys.stdin)[0]['commit']['author']['date'])")
 
-# Input file(s)
+# The output filenames
 VACCINE_PLATFORMS=owiddata/vaccine_platforms.csv
```

Comment:
Do you want to commit this output file?  We'll have to modify the workflow to add it.

Reply:
I'll update to delete it in the .sh file! It's not needed anymore